src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -9,6 +9,6 @@
         (onCancel)="router.navigate(['../'], { relativeTo: route })">
     </content-types-form>
     <div *ngIf="readyToAddFields">
-        <fields-drop-zone></fields-drop-zone>
+        <content-type-fields-drop-zone [fields]="data ? data.fields : []"></content-type-fields-drop-zone>
```

Comment:
We should do this check ```data ? data.fields : []``` inside the component, I think it's better for the component to be smart enough to check the data and react accordingly and this is something we should do always. cc: @woontsoudev